addons/mass_mailing/views/mailing_mailing_views.xml

```diff
@@ -402,6 +415,18 @@
             </field>
         </record>
 
+        <record id="view_mail_mass_mailing_calendar" model="ir.ui.view">
+            <field name="name">mailing.mailing.calendar</field>
+            <field name="model">mailing.mailing</field>
+            <field eval="2" name="priority"/>
```

Comment:
Do you really need this priority field? 
(I feel like I already asked but can't remember).

Reply:
Removed it :+1: I just noticed that most of calendar view have this attribute value.